src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -2647,9 +2634,9 @@ namespace HeatBalFiniteDiffManager {
         updatedThermalConductivity = materialDefinition->phaseChange->getConductivity(temperatureUpdated);
     }
 
-    bool findAnySurfacesUsingConstructionAndCondFD(EnergyPlusData &state, int const constructionNum)
+    bool findAnySurfacesUsingConstructionAndCondFD(EnergyPlusData const &state, int const constructionNum)
     {
-        for (auto &thisSurface : state.dataSurface->Surface) {
+        for (auto const &thisSurface : state.dataSurface->Surface) {
```

Comment:
can do this using `find_if` if you want.

Reply:
I got stuck on this one. I'm not good at forming lambdas.

Reply:
No worries. This is basically the same thing.